Classes/Notifications/NotificationCell.swift

```diff
@@ -182,6 +182,31 @@ final class NotificationCell: SelectableCell {
         watchButton.setImage(UIImage(named: "\(watchingImageName)-small")?.withRenderingMode(.alwaysTemplate), for: .normal)
 
         contentView.alpha = model.read ? 0.5 : 1
+
+        let watchAccessibilityAction = UIAccessibilityCustomAction(
+            name: model.watching ?
+                NSLocalizedString("Unwatch notification", comment: "") :
```

Comment:
Should maybe create `Constants.Strings` for these two as well? 🙂

Reply:
We could do that, but held off as they're only used once right now.